ts-tests/index.ts

```diff
@@ -195,6 +199,8 @@ manager = SDK.manager();
 // Today async clients are only possible on Node. Shared client creation not available here.
 asyncClient = AsyncSDK.client();
 asyncManager = AsyncSDK.manager();
+// Browser client for attributes binding
+browserClient = BrowserSDK.client();
```

Comment:
add `browserClient = BrowserSDK.client('a customer key');` in a new line, to check that the other `client` method signature also returns the same client type.